packages/babel-parser/src/plugins/estree.js

```diff
@@ -31,6 +33,16 @@ export default (superClass: Class<Parser>): Class<Parser> =>
       return node;
     }
 
+    estreeParseBigIntLiteral(value: any): N.Node {
+      // https://github.com/estree/estree/blob/master/es2020.md#bigintliteral
+      // $FlowIgnore
+      const bigInt = typeof BigInt !== "undefined" ? BigInt(value) : null;
```

Comment:
We should add a test case to show that the parsed result is different on node.js 6 and 8, which _does not_ have BigInt support.

https://github.com/estree/estree/blob/master/es2020.md#bigintliteral
> In environments that don't support BigInt values, value property will be null as the BigInt value can't be represented natively.

Reply:
Ah, are we not testing in older Node versions at the moment? I tested it locally with older Node versions and assumed this was already being done in CI?

Reply:
Oh, right - sorry, I was thinking of the `@babel/eslint-parser` tests. Yes, we should add a test for this in `@babel/parser`.